bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tt0123j_1l_t_5f_ckm_LO_MLM/tt0123j_1l_t_5f_ckm_LO_MLM_madspin_card.dat

```diff
@@ -0,0 +1,32 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+ set max_weight_ps_point 400  # number of PS to estimate the maximum for each event
+
+#to properly limit the number of concurrent processes for grid running
+set max_running_process 1
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+define q = u u~ d d~ c c~ s s~ b b~
+
+# specify the decay for the final state particles
+decay t > w+ b, w+ > ell+ vl
+decay t~ > w- b~, w- > q q
```

Comment:
isn't this decay mode missing the case in which t decays hadronically and t~ leptonically? shouldn't this be added (as a separate datacard, I assume) ?

Reply:
There's another sample where the lepton comes from tbar, so this should be correct

Reply:
are the datacards uploaded already?

Reply:
Yes, they are here: https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tt0123j_1l_tbar_5f_ckm_LO_MLM/tt0123j_1l_tbar_5f_ckm_LO_MLM_madspin_card.dat